components/navigation/cards.js

```diff
@@ -0,0 +1,62 @@
+import Link from "next/link";
+
+import styles from "./cards.module.scss";
+
+const NavigationCards = (props) => {
+  return (
+    <div className={styles.cardsContainer}>
+      {props.title != undefined && <h1>{props.title}</h1>}
```

Comment:
smh and you told me to not have ternary operators for clarity >:(

Reply:
jsx sux :(